tools/clang/test/CodeGenDXIL/hlsl/types/longvec-field-di.hlsl

```diff
@@ -0,0 +1,33 @@
+// RUN: %dxc -Zi -Qembed_debug -T lib_6_9 %s -DNUM=8 | FileCheck %s  --check-prefix=CHECK-LONG
+// RUN: %dxc -Zi -Qembed_debug -T lib_6_9 %s -DNUM=4 | FileCheck %s  --check-prefix=CHECK-SHORT
+
+// Test debug info for short and long vector types
+
+RWByteAddressBuffer buf;
+
+export vector<float, NUM> lv_global_arr_ret() { // expected-error{{invalid value, valid range is between 1 and 1024 inclusive}}
```

Comment:
Keeping `// expected-error{{invalid value, valid range is between 1 and 1024 inclusive}}` seems misleading, doesn't it?

Reply:
Yep, stripped that leftover from the test